### PR TITLE
Update data.md

### PR DESCRIPTION
**Pull request by mariongalley**
Update GrantNav helpsite to provide information on how to contact publishers about errors in their data

### CODE REVIEW

**Review by michaelwood — APPROVED** [review]

**Comment by mariongalley** [comment]
@michaelwood Mentioned to Matt in a planio but we'd like to hold off deploying this until the grantee amendments form is live please

**Review by mrshll1001 — DISMISSED** [review]
Approved! Ready to merge when you are.

**Review by mrshll1001 — APPROVED** [review]
Patting myself on the back by reviewing these changes but made a change to a URL as requested; instead of the absolute URL to the same page I changed it to a section based on a best-guess of the header's slug.

**Review by michaelwood — APPROVED** [review]

**Pull request merged**
